cmd/adapter/adapter.go

```diff
@@ -357,6 +360,14 @@ func main() {
 		klog.Fatalf("unable to install resource metrics API: %v", err)
 	}
 
+	// disable HTTP/2 to mitigate CVE-2023-44487 until the Go standard library
+	// and golang.org/x/net are fully fixed.
+	server, err := cmd.Server()
+	if err != nil {
+		klog.Fatalf("unable to fetch server: %v", err)
+	}
+	server.GenericAPIServer.SecureServingInfo.DisableHTTP2 = cmd.DisableHTTP2
```

Comment:
do you want to add an info log for it?

Reply:
I don't think it is necessary since the information is already in the adapter config